packaging/pkg_helpers.bash

```diff
@@ -221,6 +222,11 @@ setup_conda_pytorch_constraint() {
   else
     export CONDA_CHANNEL_FLAGS="${CONDA_CHANNEL_FLAGS} -c pytorch -c pytorch-test -c pytorch-nightly"
   fi
+
+  export CONDA_CHANNEL_FLAGS="${CONDA_CHANNEL_FLAGS} -c pytorch-nightly"
```

Comment:
Why is this change required? It seems when `PYTORCH_VERSION` is not empty, we might end up adding `pytorch-nightly` channel twice?

Reply:
Good point, I'll remove the second one